lib/Dancer2/Core/Role/Config.pm

```diff
@@ -281,8 +255,46 @@ __END__
 
 =head1 DESCRIPTION
 
-Provides a C<config> attribute that feeds itself by finding and parsing
-configuration files.
+This is the redesigned C<Dancer2::Core::Role::ConfigReader>
+to manage the Dancer2 configuration.
+
+It is now possible for user to control which B<ConfigReader>
+class to use to create the config.
+
+Use C<DANCER_CONFIG_READERS> environment variable to define
+which class or classes you want.
+
+    DANCER_CONFIG_READERS='Dancer2::ConfigReader::File::Simple Dancer2::ConfigReader::CustomConfig'
+
+If you want several, separate them with whitespace.
```

Comment:
"If you want several configuration classes, express it as a string, separated by spaces."